src/v3/src/transformer/terminal/transformTerminalTransaction.ts

```diff
@@ -237,6 +241,10 @@ export const transformTerminalTransaction = (
 
   const { messages } = transaction;
 
+  if (containsMessageKey(TERMINAL_KEY.SESSION_EXPIRED, messages)) {
+    authClient?.transactionManager.clear();
+  }
```

Comment:
For reference: https://github.com/okta/okta-signin-widget/blob/21b98876539d6c48a7108a44de337027085f41d7/src/v2/controllers/FormController.ts#L367-L370